### ISSUE TITLE
Overriding attributes for multiLineText doesn't include class attribute

### ISSUE THREAD

**Issue by benjaminkohl** [verbb/formie#1473]
### Describe the bug

In my form settings, I am attempting to override the "class" attribute for multiLineText fields with a list of classes and they do not appear on the generated textarea field when **resetClass** is _true_. I confirmed that both the "rows" and "data-test" attributes appear on the textarea element.

**Settings**
```
multiLineText: {
	fieldInput: {
		resetClass: true,
		attributes: {
			class: 'w-full bg-transparent border-black focus:border-black',
			rows: 6,
			'data-test': 'yes'
		}
	}
},
```

**Generated Element (resetClass:true)**
```
<textarea id="fui-employmentForm-zwadht-fields-message" name="fields[message]" rows="6" placeholder="Message" data-fui-id="employmentForm-message" data-test="yes"></textarea>
```

**Generated Element (resetClass:false)**
```
<textarea id="fui-employmentForm-oclmzm-fields-message" class="fui-input w-full bg-transparent !border-l-0 !border-r-0 !border-t-0 border-black focus:border-black" name="fields[message]" rows="6" placeholder="Message" data-fui-id="employmentForm-message" data-test="yes"></textarea>
```

I can't achieve the desired element styles if both the "fui-input" and our other classes are present so the desired outcome would be to remove Formie's classes and only use the classes we supply in the settings.

### Steps to reproduce

1. Outlined above.

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes ("Validate Form on Submit" is enabled)
- Custom Form Templates: No

### Craft CMS version

4.4.13

### Plugin version

2.0.29

### Multi-site?

No

**Comment by engram-design** [comment]
Just to confirm that you're using the "Input Attributes" to try and set the attributes on the `<textarea>` element. "CSS Classes" will only output on the outer field wrapper. But yes, this is indeed an issue.

Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.31"`.

**Comment by benjaminkohl** [comment]
I was using neither of those settings to set the classes. I was using the theme config settings that are passed in the second parameter of the renderForm template variable.

**Comment by JshGrn** [comment]
I am also having this issue and my example is radio buttons.

`'fieldInput' => [
                'attributes' => [
                    'class' => 'w-full px-4 py-3 text-lg lg:px-8 rounded-xl border-brand-grey text-brand-green'
                ]
            ],
            'radioButtons' => [
                'fieldOption' => [
                    'tag' => 'label',
                ],
                'fieldInput' => [
                    'resetClass' => true,
                    'attributes' => [
                        'class' => 'not-working'
                    ]
                ],
                'fieldOptionLabel' => [
                    'tag' => 'span',
                ],
            ]`

Here I would expect all inputs to have the classes under fieldInput, however when I reset them on fieldInput within radioButtons I expect the class which I then set under this to output, instead it does nothing as the classes are reset. If I don't reset then the class is appended.

I am doing this in formie.php

**Comment by engram-design** [comment]
@benjaminkohl I'm not sure I follow, sorry! That theme config code seems to work fine for me? Unless I'm missing the rest of yours for context?

@JshGrn Yeah, now that's a tricky one, because it's more of an inheritance issue. The top level `fieldInput` and `radioButtons` > `fieldInput` are merged together for the field, so when classes are reset, they reset both definitions. I'll need to look at this further.

**Comment by engram-design** [comment]
Fixed in [2.0.33](https://github.com/verbb/formie/releases/tag/2.0.33)

**Comment by WHITE-developer** [comment]
Hi @engram-design 

This "resetClass" option seems to be broken when using it in `config/formie.php` , using it in the `craft.formie.renderForm` is working fine (as long as it isn't defined in the config file).
Can you fix it there too?

--
Edit: For some reason, as soon as you add any field to the rederForm, even blank, it respects the php config. 
`{{ craft.formie.renderForm(form, { themeConfig: { repeater: {}, }}) }}`

**Comment by engram-design** [comment]
I can't replicate that sorry @WHITE-developer can you post exactly what you have in your `config/formie.php` file?

**Comment by JshGrn** [comment]
@engram-design  - As @WHITE-developer says, It doesn't work for me either in formie.php, resetClass works on its own, or attributes[class] works on its own, but both together renders no class output.

Using the 'repeater' hack above works fine - so am I using that as a workaround. 

EDIT: resetClass does not work on 'alertSuccess' either, the workaround doesn't fix that.

My config looks like this:

```
<?php

return [
    '*' => [
        'themeConfig' => [
            'resetClasses' => true,
            'fieldInputContainer' => false,
            'form' => [
                'attributes' => [
                    'id' => 'testing',
                    'class' => 'testingagain2'
                ]
            ],
            'group' => [
                'fieldLabel' => [
                    'attributes' => [
                        'class' => 'text-xl py-4 font-semibold'
                    ]
                ]
            ],
            'field' => [
                'attributes' => [
                    'class' => 'mb-4'
                ]
            ],
            'fieldInput' => [
                'attributes' => [
                    'class' => 'w-full px-4 py-3 text-lg lg:px-8 rounded-xl border-brand-grey text-brand-green'
                ]
            ],
            'fieldError' => [
                'attributes' => [
                    'class' => 'text-red-500 font-semibold mt-1'
                ]
            ],
            'alertSuccess' => [
                'resetClass' => true,
                'attributes' => [
                    'class' => 'alert-success'
                ]
            ],
            'fileUpload' => [
                'fieldInput' => [
                    'resetClass' => true,
                    'attributes' => [

                    ]
                ],
                'fieldContainer' => [
                    'attributes' => [
                        'class' => 'w-full px-4 py-3 text-lg lg:px-8 rounded-xl border bg-white border-brand-grey'
                    ]
                ],
                'fieldSummary' => [
                    'tag' => 'span',
                    'attributes' => [
                        'class' => 'text-sm text-brand-grey'
                    ]
                ],
            ],
            'agree' => [
                'fieldInput' => [
                    'resetClass' => true,
                    'attributes' => [
                        'class' => 'py-3 px-3 text-lg rounded-xl border-brand-grey text-brand-green'
                    ]
                ],
                'fieldOption' => [
                    'tag' => 'label',
                    'attributes' => [
                        'class' => 'flex items-center'
                    ]
                ],
                'fieldOptionLabel' => [
                    'tag' => 'span',
                    'attributes' => [
                        'class' => 'ml-2 mt-0.5'
                    ]
                ],
            ],
            'radioButtons' => [
                'fieldOption' => [
                    'tag' => 'label',
                    'attributes' => [
                        'class' => 'flex items-center'
                    ]
                ],
                'fieldOptions' => [
                    'attributes' => [
                        'class' => 'flex flex-col space-y-2'
                    ]
                ],
                'fieldInput' => [
                    'resetClass' => true,
                    'attributes' => [
                        'class' => 'radio'
                    ]
                ],
                'fieldOptionLabel' => [
                    'tag' => 'span',
                    'attributes' => [
                        'class' => 'ml-2 mt-0.5'
                    ]
                ],
            ],
            'submitButton' => [
                'attributes' => [
                    'class' => 'form-submit-button'
                ]
            ],

        ]
    ]
];

```

**Comment by engram-design** [comment]
Thanks for clarifying, fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.33"`.

**Comment by engram-design** [comment]
Fixed in [2.0.34](https://github.com/verbb/formie/releases/tag/2.0.34)
